concertfit/Controllers/HomeController.cs

```diff
@@ -35,12 +36,14 @@ public List<ConcertfitResponse> Get()
 
                 foreach (SpotifyArtistsResponse.TopLevel topLevel in topLevelSpotify)
                 {
-                    ConcertfitResponse concertfitResponse = new ConcertfitResponse();
-                    concertfitResponse.Artist = topLevel.Artist;
-                    concertfitResponse.Track = topLevel.Tracks;
-                    concertfitResponse.EventName = events[topLevel.Artist].Name;
-                    concertfitResponse.EventUrl = events[topLevel.Artist].Url.ToString();
-                    concertfitResponse.Dates = events[topLevel.Artist].Dates;
+                    ConcertfitResponse concertfitResponse = new ConcertfitResponse
```

Comment:
Either approach works... a bit less typing on the code review approach... and less typing is good.